guides/source/active_record_encryption.md

```diff
@@ -134,15 +134,15 @@ end
 
 You might need to ignore case when querying deterministically encrypted data. There are two options that can help you here.
 
-You can use the option `:downcase`  when declaring the encrypted attribute. This will make that content is downcased before being encrypted.
+You can use the option `:downcase` when declaring the encrypted attribute. This will downcase that content before encrypting it.
```

Comment:
```suggestion
You can use the `:downcase` option when declaring the encrypted attribute to downcase the content before encryption occurs.
```

I think we can make it even more concise! Also, we can avoid using "it," which is sometimes fairly confusing.

Reply:
I like it! Thank you :)